lib/YuiRestClient.pm

```diff
@@ -49,6 +49,10 @@ sub get_port {
     return $port;
 }
 
+sub get_timeout {
+    return ($app->{timeout}, $app->{interval});
```

Comment:
one at at time, please, it is not good practice to return two values, specially because is expected only one by the name of the function.
Besides we only need the timeout to know its current value, not the interval, which we don't modify it, and the interval depends on the test.